data/strings/sound.txt

```diff
@@ -596,6 +596,50 @@
     en = NULL
     nl = Neem
 
+  [route_recalculating]
+    en = Recalculating the route
+    af = Herbereken die roete
+    ar = إعادة حساب المسار
+    az = Marşrutun yenidən hesablanması
+    be = Пераразлік маршруту
+    bg = Преизчисляване на маршрута
+    ca = Recalcular el recorregut
+    cs = Přepočítání trasy
+    da = Genberegning af ruten
+    de = Neuberechnung der Route
+    el = Επαναϋπολογισμός της διαδρομής
+    es = Recalcular la ruta
```

Comment:
```suggestion
    es = Recalculando la ruta
```

Reply:
@zyphlar could you please apply `es` suggestions?

Reply:
@pastk done